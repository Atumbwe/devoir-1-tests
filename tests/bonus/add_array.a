2 2 5 -6 5 7 3 -6 7 3 complex +