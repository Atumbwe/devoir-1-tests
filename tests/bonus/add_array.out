 ----------
 <  22  >
 ----------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||